packages/wallets/src/features/cashier/modules/TransactionStatus/utils/transactionDetailsMappers.ts

```diff
@@ -0,0 +1,94 @@
+export const depositStatusColorMapper = {
+    CONFIRMED: 'successful',
+    ERROR: 'unsuccessful',
+    PENDING: 'warning',
+} as const;
```

Comment:
```suggestion
export const depositStatusColorMapper = {
    CONFIRMED: 'successful',
    ERROR: 'unsuccessful',
    PENDING: 'warning',
} as const;
```
I think some of these objects can be merged together. There are two variants correct - `deposit` and `withdrawal` correct?

```
const displayMapper = [{
    type: 'deposit',
    color: 'succesful',
    status: 'CONFIRMED',
    description: 'Your deposit is successful.'
}
```

Reply:
Done